### PR TITLE
fix empty messages still sending when empty

### LINKED ISSUE #684: Empty value cannot-ride-mob sends empty chat message
### Timings link

_No response_

### Expected behavior

Setting the value `cannot-ride-mob` to an empty string should not send a chat message when applicable.

### Observed/Actual behavior

Setting the value `cannot-ride-mob` to an empty string actually sends an empty chat message when applicable.

### Steps/models to reproduce

`cannot-ride-mob: ''` in config 

### Purpur version

This server is running Purpur version git-Purpur-1394 (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT) (Git: acb8a26 on ver/1.17.1)
You are running the latest version
Previous version: git-Purpur-1375 (MC: 1.17.1)

### Agreements

- [X] I am running the latest version of Purpur available from https://purpur.pl3x.net/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Purpur.

### Other

Issue #53 suggests this was previously possible.

### PR DESCRIPTION
**Pull request by Oharass**
Fix #684. Use empty message for cannot-ride-mob. This adds

### CODE REVIEW

**Review by katekerllenevich — APPROVED** [review]

**Review by BillyGalbreath — CHANGES_REQUESTED** [review]
While this would fix it for this specific option, it won't for any of the others. A better way is to check in the `sendMessage` method. Seems like when I switched everything to components that logic was lost somewhere.

**Comment by Oharass** [comment]
ok. I will fix this.

**Comment by Oharass** [comment]
I think this is correct now. Fixed for all. I use "isEmpty" instead of "isBlank" so you can still send blank message if you like with " ". This adds.

File: patches/server/0006-Component-related-conveniences.patch [hunk 1]
```diff
@@ -116,7 +116,7 @@ index 732eb171f6a8fe1b02044a00fcf85217c41116ac..3b7d72f035d7fd74971af043294047ff
 +    }
 +
 +    public void sendMessage(@Nullable String message, UUID sender) {
-+        if (message != null) {
++        if (org.apache.commons.lang3.StringUtils.isNotEmpty(message)) {
```
Comment (encode42) [inline patches/server/0006-Component-related-conveniences.patch:119]: Why use StringUtils instead of Java's native `String#isEmpty`?

Comment (Oharass) [reply patches/server/0006-Component-related-conveniences.patch:119]: It handles null. I may also do `message != null && !message.isEmpty()`

Comment (encode42) [reply patches/server/0006-Component-related-conveniences.patch:119]: Ah, I see. Was just wondering

**Review by encode42 — CHANGES_REQUESTED** [review]
LGTM, just resolve conflicts

**Pull request closed without merging**